### ISSUE TITLE
[0.8.4] B-DropDown: Position "different" behaviour

### ISSUE THREAD

**Issue by susanne99** [bootstrap-vue/bootstrap-vue-next#1049]
first of all, in version 0.7.4 it works !!!

according to bootstrap vue docu:
_The dropdown menu can either be left aligned (default) or right aligned .........._
i use the default setting, that means i do not set the "left" prop explicit.
that works great til version 0.7.4.

in version 0.8.4 there is a different behaevoir. the default value is not "left" and i cannot set "left"
i want the menu below, as in version 0.7.4

```
 <b-dropdown
       split
       split-variant="outline-secondary"
       variant="secondary"
       :text="searchtypetext"
       size="sm"
       class="mx-1 text-nowrap">
       <!-- Leider funktionieren data-attributes (noch) nicht, also muss ich function parameter dafür verwenden -->
       <b-dropdown-item @click="onclick_searchtype($event, 'lastname')"  href="#" data-searchtype="lastname">
                   Nachname oder Versicherungsnummer</b-dropdown-item>
       <b-dropdown-item @click="onclick_searchtype($event, 'birthdate')"  href="#" data-searchtype="birthdate">
                   Geburtsdatum</b-dropdown-item>
</b-dropdown>
```

good in 0.7.4 :)
![dropdown_0_7_3 ](https://user-images.githubusercontent.com/112922199/232067936-12adce2e-45c0-4c5f-aeeb-12d8f04edaa6.JPG)

Not good in 0.8.4  :(
![dropdown](https://user-images.githubusercontent.com/112922199/232067943-ec774069-01e9-47c7-b78d-ce6655f79dce.JPG)

**Comment by VividLemon** [comment]
I'd recommend reading the breaking issues list in 0.8

**Comment by susanne99** [comment]
Q: How can i position the menu below ?

**Comment by susanne99** [comment]
you are right. split was the problem - it works now right in 0.8.4

**Comment by susanne99** [comment]
fixed in 0.8.4
